packages/block-editor/src/components/block-switcher/block-transformations-menu.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { MenuGroup, MenuItem } from '@wordpress/components';
+
+const BlockTransformationsMenu = ( {
+	possibleBlockTransformations,
+	onSelect,
+} ) => {
+	return (
+		<MenuGroup>
```

Comment:
`MenuGroup` takes `label` prop. The way it is implemented now, it won't be announced by screen readers.

https://github.com/WordPress/gutenberg/tree/master/packages/components/src/menu-group#development-guidelines

Reply:
Thanks! I added both labels.